packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -877,6 +877,7 @@ class Scaffold extends StatefulWidget {
     this.primary = true,
     this.drawerDragStartBehavior = DragStartBehavior.start,
     this.extendBody = false,
+    this.scrimColor,
```

Comment:
Looks like you missed refactoring all uses of drawerScrimColor

Reply:
sorry my bad! done